### ISSUE TITLE
Disable account creation (not just via access question)

### ISSUE THREAD

**Issue by jbattat** [jgm/gitit#532]
I'd like to disable account creation via the web interface.  I know about the access question, and I currently use it, but I'd prefer to fully disable account creation, rather than just making it probable that a random user can't create an account.  

As far as I know, it's not possible to disable account creation (according to mailing list discussions from ~2010).  If this is still the case, I'd like to request that this feature be added.

Many thanks for a great tool!

**Comment by jgm** [comment]
You can always put the account creation page under HTTP
authentication.

+++ jbattat [Jan 06 16 06:59 ]:

**Comment by jbattat** [comment]
True, and I had seen that suggestion in the mailing lists.  It just seems cleaner to avoid the issue entirely by disabling the "Account Creation" option.  But if this is a change that seems simple, but is in fact rather involved to implement, then I will go one of the other two routes (access question and http authentication).  -- Thanks again
